src/components/FiltersBar/FiltersBar.js

```diff
@@ -1,60 +1,107 @@
-import React from "react";
-import styles from "./FiltersBar.css";
-import MovieFilters from "../MovieFilters/MovieFilters";
-import MovieSort from "../MovieSort/MovieSort";
+import React, { Component } from 'react';
+import styles from './FiltersBar.css';
+import MovieFilters from '../MovieFilters/MovieFilters';
+import MovieSort from '../MovieSort/MovieSort';
 
 const movieFilters = [
   {
     id: 10,
     isActive: true,
-    title: "All",
+    title: 'All',
   },
   {
     id: 11,
     isActive: false,
-    title: "Documentary",
+    title: 'Documentary',
   },
   {
     id: 12,
     isActive: false,
-    title: "Comedy",
+    title: 'Comedy',
   },
   {
     id: 13,
     isActive: false,
-    title: "Horror",
+    title: 'Horror',
   },
   {
     id: 14,
     isActive: false,
-    title: "Crime",
+    title: 'Crime',
   },
 ];
 
 const movieSort = [
   {
     id: 15,
     isActive: true,
-    title: "Sort by",
+    title: 'Sort by',
   },
   {
     id: 16,
     isActive: false,
-    title: "elease Date",
+    title: 'elease Date',
   },
 ];
 
-const FiltersBar = () => {
-  return (
-    <div className={styles.wrapper}>
-      <div className={styles.movieFilters}>
-        <MovieFilters tabs={movieFilters} />
-      </div>
-      <div className={styles.movieFilters}>
-        <MovieSort tabs={movieSort} />
-      </div>
-    </div>
-  );
-};
+export default class FiltersBar extends Component {
+  constructor() {
+    super();
+
+    this.defaultTitle = 'select sort';
+
+    this.state = {
+      title: this.defaultTitle,
```

Comment:
It is better to use default props https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values